stackdriver.go

```diff
@@ -253,10 +285,45 @@ func NewExporter(o Options) (*Exporter, error) {
 		}
 		o.ProjectID = creds.ProjectID
 	}
+	if o.Location == "" {
+		ctx := o.Context
+		if ctx == nil {
+			ctx = context.Background()
+		}
+		zone, err := metadataapi.Zone()
+		if err != nil {
+			log.Printf("Setting Stackdriver default location failed: %s", err)
+		} else {
+			log.Printf("Setting Stackdriver default location to %q", zone)
```

Comment:
Just checking. Is this a leftover debugging aid or intended?

Reply:
I put this in intentionally.